WSO2Scenariofile.yaml

```diff
@@ -0,0 +1,18 @@
+﻿scenarios:
+  -
+    name: 'Multi-factor authentication for WSO2 Identity Server Management Console'
```

Comment:
Please include all 30 scenario's names and description here.

Reply:
Added all the scenarios as requested